### ISSUE TITLE
Qobuz - KeyError: 'songinfo_loop'

### ISSUE THREAD

**Issue by adi1** [mavit/slimpris2#18]
There are certain tracks on Qobuz which cause an error:

```
DEBUG: query: ['songinfo', '0', None, 'tags:acdgijkKlmOtuwy', 'track_id:-164612608'], result: {}

Traceback (most recent call last):
  File "/usr/local/bin/slimpris2", line 1345, in receive_comet_responses
    sb_wrapper.monitor(info)
  File "/usr/local/bin/slimpris2", line 407, in monitor
    metadata = self._dbus_service.update_property(
  File "/usr/local/bin/slimpris2", line 730, in update_property
    value = getter()
  File "/usr/local/bin/slimpris2", line 593, in __get_metadata
    metadata = lms.metadata(track['id'])
  File "/usr/local/bin/slimpris2", line 1008, in metadata
    return songinfo['songinfo_loop']
KeyError: 'songinfo_loop'
```

I can't really find the pattern, but some tracks don't seem to return songinfo

When it works:
```
DEBUG: query: ['songinfo', '0', None, 'tags:acdgijkKlmOtuwy', 'track_id:-256009592'], result: {'songinfo_loop': [{'id': '-256009592'}, {'title': 'Gone'}, {'artist': 'Jorja Smith'}, {'coverid': '-256009592'}, {'duration': '195'}, {'coverart': '0'}, {'artwork_url': '/imageproxy/https%3A%2F%2Fstatic.qobuz.com%2Fimages%2Fcovers%2Fjb%2F2q%2Fnp4uf025v2qjb_600.jpg/image.jpg'}, {'album': 'Be Right Back'}, {'url': 'qobuz://119683533.flac'}, {'year': '0'}]}
```
not working:
```
DEBUG: query: ['songinfo', '0', None, 'tags:acdgijkKlmOtuwy', 'track_id:-164612608'], result: {}
```

Would it be possible to degrade nicely when the metadata is not available, to still allow next/prev/play/pause controls to keep working? Right now, the only way to get the controls working again is to use a different interface to go to another track, or just wait for a track with metadata to come up.

**Comment by mavit** [comment]
I don't use Qobuz myself, but I think https://github.com/mavit/slimpris2/commit/9421365e05c4e4e9cceaab83d8b184199a3cb30a should do the trick.  Could you give it a try and let me know?

**Comment by adi1** [comment]
I finally got around to testing it, sorry for the late update.

Yes, this commit fixes the issue.

**Comment by mavit** [comment]
Fixed in 3.0.1.
